### PR TITLE
Combinate cells

### PR DESCRIPTION
**Pull request by AngheloZambrana**

### CODE REVIEW

File: src/Rendering.hs [hunk 1]
```diff
@@ -2,32 +2,48 @@ module Rendering where
...
+    translatedText = translate (fromIntegral x * cellWidth + cellWidth / 2 -20 ) (fromIntegral (n - y) * cellHeight - cellHeight / 2 -20) $ scale 0.5 0.5 $ boldText 1.4 $ color black $ text (show value)
+
 -- GRID DRAWING
 boardGridColor :: Color
 boardGridColor = makeColorI 0 0 0 255 
 
--- combines all the lines of the board into a single picture
--- where drawlines generate a pair of lines for each valui in the list
+-- Combina todas las líneas del tablero en una sola imagen
+-- donde drawLines genera un par de líneas para cada valor en la lista
```
Comment (DeidamiaFuentes) [inline src/Rendering.hs]: Why did you translate the comments?

File: src/Rendering.hs [hunk 2]
```diff
@@ -2,32 +2,48 @@ module Rendering where
...
-drawCell (y, x) (Ocuppied value) = pictures [ coloredRectangle, translatedText ]
-  where
-    coloredRectangle = translate (fromIntegral x * cellWidth + cellWidth / 2) (fromIntegral (n-y) * cellHeight - cellHeight / 2) $ color red $ rectangleSolid cellWidth cellHeight
-    translatedText = translate (fromIntegral x * cellWidth + cellWidth / 2 -20 ) (fromIntegral (n-y) * cellHeight - cellHeight / 2 -20) $ scale 0.5 0.5 $ boldText 1.4 $ color white $ text (show value)
+                  , line [(0.0, i * cellHeight), (fromIntegral screenWidth, i *  cellHeight)]]
 
--- Iteratates throught the board to draw each cell and combines it with the board grid 
--- as one single picture
+-- Itera por el tablero para dibujar cada celda y las combina con la cuadrícula del tablero
+-- como una sola imagen
```
Comment (DeidamiaFuentes) [inline src/Rendering.hs]: Why did you translate the comments?

File: src/Rendering.hs (hunk 1 above)
Comment (AngheloZambrana) [reply src/Rendering.hs]: A little confusion, I commit the changes with the traductions

File: src/Rendering.hs (hunk 2 above)
Comment (AngheloZambrana) [reply src/Rendering.hs]: A little confusion, I commit the changes with the traductions

**Review by DeidamiaFuentes — APPROVED** [review]
Everything works grate c:

**Review by MayerliSantander — APPROVED** [review]

**Pull request merged**